### DISCUSSION TITLE
[Feature Request] Block miniplayer autopause for "for kid" marked video

### DISCUSSION THREAD

**Discussion by CDzungx** [ReVanced/revanced-discussions#323] [Question] [answered]
When minimizing the video marked as kid video (even they aren't), the video will be paused, user must open the video tab to continue. Kinda annoying, so please add a patch block this thing.
![Picsart_22-07-16_17-44-59-878](https://user-images.githubusercontent.com/59353870/179351784-c0071932-d2de-42a3-ac76-294470a12eee.png)

**Comment by oSumAtrIX** [comment, marked as answer]
Dupe of #23.

**Reply by CDzungx** [reply]
Is it? This is miniplayer autopause tho? it isn't playlist thinggy

**Reply by oSumAtrIX** [reply]
Then move to that discussion and add your suggestion.

**Reply by ghost** [reply]
This mod is already been added on Vanced 17.03.38.

Just need to be ported to ReVanced.

It's a patch related to MinimizedPlaybackController.smali.

**Reply by ponces** [reply]
I'll do it ;)

**Reply by oSumAtrIX** [reply]
IIRC we already do whatever Vanced did in the class `MinimizedPlaybackController`. @ponces But confirm to be sure.

**Reply by ghost** [reply]
> IIRC we already do whatever Vanced did in the class `MinimizedPlaybackController`. @ponces But confirm to be sure.

Wdym?

**Reply by oSumAtrIX** [reply]
There is already a patch working on `MinimizedPlaybackController`.

**Reply by ponces** [reply]
I can confirm it tomorrow. I committed to this because I started today looking into the same topic but on Music. If I'm successful with it, I think I can also make it on YT. Knowing that it's already a feature of Vanced just makes my job easier I guess :)

**Reply by ghost** [reply]
> There is already a patch working on `MinimizedPlaybackController`.

So this thread could be marked as completed.

**Reply by oSumAtrIX** [reply]
Well "kids" videos still are still not handled.

**Reply by ghost** [reply]
> Well "kids" videos still are still not handled.

There's no much things to mod on "kids video". Just the "save to playlist" button.

**Reply by oSumAtrIX** [reply]
@OxrxL RE: https://github.com/TeamVanced/Vanced/issues/913#issuecomment-860325040

**Reply by ghost** [reply]
> @OxrxL RE: [TeamVanced/Vanced#913 (comment)](https://github.com/TeamVanced/Vanced/issues/913#issuecomment-860325040)

Yeah...this is currently the unique missing mod (related to "kids video") on ReVanced.

But this should be discussed on another thread I guess (I opened a new one last day).

**Reply by ponces** [reply]
@oSumAtrIX 
I may be wrong but I can't find the patch on `MinimizedPlaybackPolicyController` that you are talking about...
Currently there is no patch unblocking the minimized player on Kids videos as OP said.
I'm currently making one based on `MinimizedPlaybackPolicyController` which should work similar to the patch that I made on Music.
Hopefully, both of these changes will be included on mine next PR :)

**Reply by ponces** [reply]
Just pushed https://github.com/revanced/revanced-patches/pull/200 and https://github.com/revanced/revanced-patches/pull/201!
